ccmlib/scylla_docker_cluster.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, *args, **kwargs):
         self.docker_id = None
         self.local_data_path = os.path.join(self.get_path(), 'data')
         self.local_yaml_path = os.path.join(self.get_path(), 'conf')
-        self.docker_name = f'{self.cluster.name}-{self.name}'
+        self.docker_name = f'{self.cluster.get_path().split("/")[-1]}-{self.cluster.name}-{self.name}'
```

Comment:
@fgelcer 
What the result of `self.cluster.get_path().split("/")[-1]}`?

Reply:
if running from ccm (like command line) it will be the `cluster_name`
if running from dtest, then it will be the test directory with a unique name